app/dashboards/educator_dashboard.rb

```diff
@@ -8,35 +9,26 @@ class EducatorDashboard < Administrate::BaseDashboard
   # which determines how the attribute is displayed
   # on pages throughout the dashboard.
   ATTRIBUTE_TYPES = {
-    school_id: SchoolNameField.with_options(searchable: false),
-    homeroom: HomeroomNameField.with_options(searchable: false),
-    students: Field::HasMany.with_options(searchable: false),
-    interventions: Field::HasMany.with_options(searchable: false),
-    id: Field::Number.with_options(searchable: false),
-    email: Field::String.with_options(searchable: false),
-    encrypted_password: Field::String.with_options(searchable: false),
```

Comment:
These seem scary but most haven't existed on the model in a long time.  Without digging into Administrate all the way, these appear to be only limiting what options are available for other config, based on how this behaved editing them here.

That being said, another audit pass on administrate would be a good security step.